components/scream/cime_config/config_compsets.xml

```diff
@@ -41,7 +41,7 @@
 
   <compset>
       <alias>F2000-SCREAMv1-AQP1-noAero</alias> <!-- atm aquaplanet compset (using SCREAMv1), 2000 initial conditions, turn of Aerosol forcing -->
-      <lname>2000_SCREAM%no_Aero%AQUA_SLND_SICE_DOCN%AQP1_SROF_SGLC_SWAV</lname>
+      <lname>2000_SCREAM%noAero%AQUA_SLND_SICE_DOCN%AQP1_SROF_SGLC_SWAV</lname>
```

Comment:
This is probably ok for the compset parser but deviates from the typical format of time_comp%phys_comp%phys. When there are multiple phys options, usually they are split by hyphen. The order does not really matter as long as accordingly in pattern matching. In existing compsets, usually AQUA goes before the others, such as _SCREAM%AQUA-noAero.

Reply:
I can fix that